storage-resize-images/extension.yaml

```diff
@@ -247,8 +247,7 @@ params:
   - param: SHARP_OPTIONS
     label: Sharp constructor options for resizing images
     description: >
-      Provide a optional output option stringified object containing Sharp Output Options for selected image types conversion.
-      eg. `{ failOnError: false, limitInputPixels: true }`
+      Provide an optional output option stringified object containing Sharp Output Options for selected image types conversion. eg. `{ "failOnError": false, "limitInputPixels": true, "fit": "inside" }` The `"fit": "inside"` option ensures the image fits within given dimensions, maintaining aspect ratio, scaling down as needed without cropping or distortion. Learn more about [`Sharp constructor options`](https://sharp.pixelplumbing.com/api-resize#resize).
```

Comment:
The description here is inaccurate, it's not "output options", try to make it more clear